File: main/contrib/Windows API Code Pack 1.1/source/Samples/Shell/StarBackupUsingLibraries/CS/Wizard/WizardReturnEventArgs.cs
//Copyright (c) Microsoft Corporation.  All rights reserved.

using System;
using System.Collections.Generic;
using System.Text;

namespace Microsoft.WindowsAPICodePack.Samples.StarBackupSample
{
    public class WizardReturnEventArgs
    {
        WizardResult result;
        object data;

        public WizardReturnEventArgs(WizardResult result, object data)
        {
            this.result = result;
            this.data = data;
        }

        public WizardResult Result
        {
            get { return this.result; }
        }

        public object Data
        {
            get { return this.data; }
        }
    }
}

File: main/contrib/Windows API Code Pack 1.1/source/Samples/Shell/StarBackupUsingLibraries/VB/Wizard/WizardReturnEventArgs.vb
'Copyright (c) Microsoft Corporation.  All rights reserved.


Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text

Namespace Microsoft.WindowsAPICodePack.Samples.StarBackupSample
	Public Class WizardReturnEventArgs
        Private result_Renamed As WizardResult
        Private data_Renamed As Object

		Public Sub New(ByVal result As WizardResult, ByVal data As Object)
			Me.result_Renamed = result
			Me.data_Renamed = data
		End Sub

		Public ReadOnly Property Result() As WizardResult
			Get
				Return Me.result_Renamed
			End Get
		End Property

		Public ReadOnly Property Data() As Object
			Get
				Return Me.data_Renamed
			End Get
		End Property
	End Class
End Namespace
